### DISCUSSION TITLE
Forced autoplay mode and esc button won't work

### DISCUSSION THREAD

**Discussion by utku-helvaci** [ppy/osu#25659] [Q&A] [answered]
### Type

Game behaviour

### Bug description

For couple versions and 2023.1130.0-1 linux appimage, maps always load with autoplay mode enabled even though in the mods its disabled. Enabling incompatible mods like "cinema" mode doesn't fix the issue as game enables autoplay mode anyways. Pressing esc does nothing and only way to exit the game is alt+f4

### Screenshots or videos

[recording from fresh config](https://youtu.be/PfSkJGmxh9c)

### Version

2023.1130.0-1

### Logs

[runtime.log](https://github.com/ppy/osu/files/13540177/runtime.log)
[performance.log](https://github.com/ppy/osu/files/13540178/performance.log)
[network.log](https://github.com/ppy/osu/files/13540179/network.log)
[input.log](https://github.com/ppy/osu/files/13540180/input.log)
[database.log](https://github.com/ppy/osu/files/13540181/database.log)
[fresh osu! folder](https://github.com/utku-helvaci/files/tree/1ade313e4603f4cf21576b895810726bfb5ba313/2023.1130.0-1%20linux%20appimage%2C%20arch%20linux%2C%20forced%20autoplay%20bug)

**Comment by bdach** [comment, marked as answer]
Are you pressing control? Are you remapping keys in some way? What keyboard layout are you using?

**Reply by utku-helvaci** [reply]
That seems to be the issue. I've disabled keymaps on opentabletdriver and that fixed it after a reboot. Thank you!
